docs/development.md

```diff
@@ -124,13 +117,17 @@ using:
 
 ```bash
 make clean # Clean up any previous manifests
+make clean-examples # Clean up any previous example manifests
```

Comment:
this isn't strictly needed since `make generate-examples` runs it already.

Reply:
👍